### ISSUE TITLE
Unusable on Android

### ISSUE THREAD

**Issue by trops** [jshanson7/react-native-swipeable#73]
I have implemented this library in a project and while the iOS works perfectly the android application is virtually unusable. I have two right buttons, and when I attempt to swipe to reveal the buttons it takes about 5-6 seconds before the buttons are revealed. I have to hold my finger in position and not release or the buttons are not revealed whatsoever. I have the Swipeable component in a FlatList and am managing scrollEnabled while swiping.

Has anyone had a similar experience? I cannot use this library without this resolved.

**Comment by gams95** [comment]
Is debug js remotely on? if yes, try to turn it off. It slows down animation whenever debugger is on

**Comment by Under-Warz** [comment]
Yes me too, I think it's because the missing of the Animated warning : Animated: `useNativeDriver` was not specified. This is a required option and must be explicitly set to `true` or `false`

Please fix conflict and merge this PR : https://github.com/jshanson7/react-native-swipeable/pull/101 guys :)
